config/deploy/staging.rb

```diff
@@ -7,6 +7,12 @@
 set :deploy_to, ENV.fetch('DIR', '/opt/heliotrope')
 server "#{ENV.fetch('USER')}@#{ENV.fetch('HOST')}", roles: [:web, :app, :db, :puma, :resque_pool]
 
+# ensure umask is set to allow groups to read/write
+SSHKit.config.umask = "0002"
+
+# prevent permissions issues by using a temp dir under user's home dir
+set :tmp_dir, ENV['TMP'] || '/tmp'
```

Comment:
Aaron suggests setting the `:tmp_dir` inside of `/tmp` because we may run into performance issues with AFS. So you could re-use the `$USER` variable we are already passing and change this line to something like:

```
set :tmp_dir, '/tmp' || ENV['USER']
```